app/Http/Controllers/MappingController.php

```diff
@@ -165,4 +170,62 @@ public function destroy(Webhook $webhook, Mapping $mapping)
             ]);
         }
     }
+
+    public function import(Webhook $webhook, MappingImportRequest $request)
+    {
+        try {
+            $keys = $this->mappingRepository->getKeys($webhook);
+            $data = json_decode(file_get_contents($request->file), true);
+
+            Validator::make($data, [
+                "*.key" => "required|max:100",
+                "*.value" => "required|max:100",
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/MappingController.php</br>**Line**: 182</br>String "*.value" does not require double quotes; use single quotes instead